plugins/http_plugin/http_plugin.cpp

```diff
@@ -437,13 +437,12 @@ class http_plugin_impl : public std::enable_shared_from_this<http_plugin_impl> {
             
             add_api({{
                std::string("/v1/node/get_supported_apis"),
-               [&](const string&, string body, url_response_callback cb) mutable {
+               [&](string, string body, url_response_callback cb) {
```

Comment:
I'm curious why this change?

Reply:
That was to match the definition of the handler (see https://github.com/AntelopeIO/leap/blob/558f69064b9dbcbb2c3de8f434bd75a1d33bb2ab/plugins/http_plugin/include/eosio/http_plugin/http_plugin.hpp#L29).

I probably should change that instead. Really it should be a `std::string_view` IMO.

Reply:
Actually maybe these should be `std::string` values, because they are `std::move`d and posted to different threads [here](https://github.com/AntelopeIO/leap/blob/558f69064b9dbcbb2c3de8f434bd75a1d33bb2ab/plugins/http_plugin/http_plugin.cpp#L102-L110)

Reply:
I'm thinking of leaving it as is for now. I think we should change the api to:

`using url_handler = std::function<void(string&&,string&&,url_response_callback&&)>; `

Reply:
In a [separate branch](https://github.com/AntelopeIO/leap/commit/7d8b3265eb77c7fcddf279cd6be6c5081da87f3c), I have updated the url_handler to take rvalues. What do you guys think?

Reply:
I say include that in this PR.

Reply:
@heifner @mikelik @linh2931  I have updated the `url_handler` type and API definitions to use rvalues. Would you guys like to have another look?